src/components/App/App.js

```diff
@@ -32,19 +32,23 @@ onClickDone = id => {
     if (item.id ===id){
       newItem.isDone = !item.isDone;
     }
-
     return newItem;
   });
 
   this.setState({ items: newItemList });
 };
 
+onClickDelete = id => {
+  const newList = this.state.items.filter(item => item.id != id);
```

Comment:
Codacy found an issue: [Expected '!==' and instead saw '!='.](https://app.codacy.com/gh/WebHeroSchool/evafinal/pullRequest?prid=5910854)